spec/models/article_spec.rb

```diff
@@ -234,6 +234,18 @@ def build_and_validate_article(*args)
       end
     end
 
+    describe "title validation" do
+      let(:title) { "U+202D" }
+
+      let!(:test_article) { build(:article, title: title) }
+
+      before { test_article.validate }
+
+      it "produces a proper title" do
+        expect(test_article.title).to eq(title)
```

Comment:
This is correct. A title shouldn't contain unicode characters but should show the unicode in case someone wants a title like this: "How to strip the U+202D character from a string in Ruby"

Reply:
I think we're missing the other path of validation logic, when the validation needed to modify the title did that happen?